blog.html

```diff
@@ -10,16 +10,70 @@
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,300,600,800' rel='stylesheet' type='text/css'>
 
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1 class="logo">Relaxr</h1>
+				<nav>
+					<ul class="clearfix head-nav">
+						<li>About</li>
+						<li>FAQ</li>
+						<li>Team</li>
+						<li>Contact Us</li>
+						<li>Blog</li>
+					</ul>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<div class="clearfix articles">
+				<main class="clearfix">
+					<article>
+						<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+						<img src="images/blog_photo1.jpg" alt="Dragon Boat Racing">
+						<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+						<h5>Read more ></h5>
+						<hr>
+					</article>
+						<h2>I traveled across the globe thanks to Relaxr's Excel automation</h2>
+						<img src="images/blog_photo2.jpg" alt="Canoes in a River">
+						<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+						<h5>Read more ></h5><br>
+						<hr>
+					</article>
+				</main>
+				<aside class="clearfix">
+					<div class="side-categories">
+						<h3>Categories</h3>
+						<ul class="aside-nav">
+							<li>Success stories</li>
+							<li>Stats</li>
+							<li>How tos</li>
+							<li>Best of</li>
+							<li>Business tips</li>
+						</ul>
+					</div>
+					<div class="side-about-relaxr">
+						<h3>About Relaxr</h3>
+						<p class="about"> Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttior, sapien sed tincidunt fringilla.</br>
+						<strong>Learn More</strong></p>
+					</div>
+					<div class="clearfix ad-unit">
+						<h3>Ad Unit</h3>
+						<section class="ad">
```

Comment:
You probably wanted your `h3` inside of your section here. You've got the `h3` and the `section` inside of your `div` but only the `section` gets a white background color.